x-pack/plugins/profiling/public/views/storage_explorer/summary.tsx

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiLink, EuiPanel, EuiStat, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { asDynamicBytes } from '@kbn/observability-plugin/common';
+import React from 'react';
+import { StackTracesDisplayOption, TopNType } from '../../../common/stack_traces';
+import { StorageExplorerSummaryAPIResponse } from '../../../common/storage_explorer';
+import { useProfilingDependencies } from '../../components/contexts/profiling_dependencies/use_profiling_dependencies';
+import { LabelWithHint } from '../../components/label_with_hint';
+import { useProfilingParams } from '../../hooks/use_profiling_params';
+import { useProfilingRouter } from '../../hooks/use_profiling_router';
+import { asPercentage } from '../../utils/formatters/as_percentage';
+
+interface Props {
+  data?: StorageExplorerSummaryAPIResponse;
+  isLoading: boolean;
+}
+
+interface SummaryInfo {
+  title: string;
+  value?: string | number;
+  hint?: string;
+}
+
+export function Summary({ data, isLoading }: Props) {
+  const { query } = useProfilingParams('/storage-explorer');
+  const { rangeFrom, rangeTo, kuery } = query;
+  const profilingRouter = useProfilingRouter();
+  const {
+    start: { core },
+  } = useProfilingDependencies();
+
+  const summaryInfo: SummaryInfo[] = [
+    {
+      title: i18n.translate('xpack.profiling.storageExplorer.summary.totalData', {
+        defaultMessage: 'Total data',
+      }),
+      value: data?.totalProfilingSizeBytes
+        ? asDynamicBytes(data?.totalProfilingSizeBytes)
+        : undefined,
+      hint: i18n.translate('xpack.profiling.storageExplorer.summary.totalData.hint', {
+        defaultMessage:
+          'Total Storage Size of all Profiling Indices including replicas, ignoring the filter settings',
```

Comment:
```suggestion
          'Total storage size of all Universal Profiling indices including replicas, ignoring the filter settings',
```

I used the longer official name (Universal Profiling) here and made minor corrections to keep capitalization consistent.